front/cypress/e2e/project_page_timeline.cy.ts

```diff
@@ -148,29 +148,42 @@ describe('New timeline project', () => {
       .contains(phaseCurrentTitle);
     cy.get('.e2e-project-process-page').contains(phaseLongDescription);
 
+    // Verify that the description is collapsed by default
     cy.get('#e2e-project-phase-description-see-less-button').should(
       'not.exist'
     );
+
+    // Read more/less button functionality
     cy.get('#e2e-project-phase-description-read-more-button')
-      .should('exist')
+      .should('be.visible')
       .click();
-
     cy.get('#e2e-project-phase-description-read-more-button').should(
       'not.exist'
     );
+
     cy.get('#e2e-project-phase-description-see-less-button')
-      .should('exist')
-      .click();
+      .should('be.visible')
+      .then(($btn) => {
+        cy.wrap($btn).click();
+      });
```

Comment:
This stabilizes the test. The best explanation I can find is that between asserting should('be.visible') and clicking the button, there is React/DOM stuff going on (because of the focus operation in ReadMoreWrapper?) which causes the button to be non-interactable for short bit. However, I've also improved that code in ReadMoreWrapper (using refs + setTimeout), but it was not enough to make the test stable.

Reply:
So, frustratingly, no general learnings to take to the team about this one. :/

Reply:
In a lot of those cases it's a combination of factors and it's sometimes difficult to pinpoint the exact reason. Totally agree it's frustrating!